runtime/doc/lua.txt

```diff
@@ -648,6 +648,13 @@ regex:match_line({bufnr}, {line_idx} [, {start}, {end}])  *regex:match_line()*
     |regex:match_str()|. If {start} is used, then the returned byte indices
     will be relative {start}.
 
+------------------------------------------------------------------------------
+VIM.LPEG                                                            *lua-lpeg*
+
+                                                             *vim.lpeg* *vim.re*
+The Lpeg library for parsing expression grammars is being included as
+`vim.lpeg`. In addition, its regex-like interface is available as `vim.re`.
```

Comment:
```suggestion
`vim.lpeg`. In addition, its regex-like interface is available as `vim.re`, see https://www.inf.puc-rio.br/~roberto/lpeg/re.html for documentation.
```

Or do we want to vendor that as well?

Reply:
maybe but it could be a followup. adding the links for now.

Reply:
(I looked at the docs; they're not huge, so pulling them in as `lua.txt` is perfectly feasible, and possibly useful -- depending on how useful `vim.lpeg` and `vim.re` is for users, of course. But as you say, possible followup, and good-first-issue.)